src/components/Button/Icon.tsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
 
 // eslint-disable-next-line react/prop-types
```

Comment:
hmm not sure, but I think that must have been why we added it. When I remove it now, I have the error `'width' is missing in props validation - eslint(react/prop-types)`

Reply:
but, i guess that is just a visual error  since this icon won't be used in other repositories and is just for demoing. I just assumed that it would cause a type error as well